gm4_blast_furnaces/data/blast_furnaces/functions/update.mcfunction

```diff
@@ -0,0 +1,7 @@
+# Called from main.mcfunction
```

Comment:
`update` is a poor name for this function as it is already used to refer to the module installation sequence. `process` (or similar term) would be a more appropriate name

Reply:
I asked @Misode for a name suggestion. He suggested `update`.